src/components/IGVBrowser.js

```diff
@@ -66,8 +74,12 @@ const IGVBrowser = _.flow(
             }, selectedFiles))
           }
 
+          const customReference = _.find({ name: refGenome }, customReferences)
+
+          const mergedOptions = customReference ? _.merge(options, customReference.referenceDefinition) : options
```

Comment:
This works. My only thought is that I had to run an experiment (in a console, though it could have been in my mind if I was more awake and had more recent experience with `_.merge`) to see the final shape of `mergedOptions`. An alternative way of expressing this might be (untested, but I think it works; also incorporating my other suggestion above):
```
// Additional references supported by Terra that are not included in IGV
const customReferences = {
  'MN908947.3': {
    id: 'sarsCov2RefId.3', indexed: false,
    fastaURL: 'https://storage.googleapis.com/gcp-public-data--broad-references/sars-cov-2/MN908947.3/nCoV-2019.reference.fasta'
  }
}

...

    const customReference = customReferences[refGenome]
    const options = {
      genome: refGenome,
      reference: customReference,
      tracks: { ... }
    }
```
No `_.merge` needed. What do you think?

Reply:
I think this is nicer, thanks for this! I've incorporated it into 45a72a6241aadd63636b436808b6d14a5eb8d86d

Reply:
(and btw I tested the suggestion and it does indeed work)